<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgToolCrop</class>
 <widget class="QWidget" name="WdgToolCrop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crop</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item column="1" row="2">
          <widget class="QCheckBox" name="boolCenter">
           <property name="toolTip">
            <string>Resize keeping center fixed</string>
           </property>
           <property name="text">
            <string>Center</string>
           </property>
          </widget>
         </item>
         <item column="0" row="1">
          <widget class="QLabel" name="label_vertiPos">
           <property name="toolTip">
            <string>Vertical Position</string>
           </property>
           <property name="text">
            <string>&amp;Y:</string>
           </property>
           <property name="buddy">
            <cstring>intY</cstring>
           </property>
          </widget>
         </item>
         <item column="0" row="0">
          <widget class="QLabel" name="label_horizPos">
           <property name="toolTip">
            <string>Horizontal Position</string>
           </property>
           <property name="text">
            <string>&amp;X:</string>
           </property>
           <property name="buddy">
            <cstring>intX</cstring>
           </property>
          </widget>
         </item>
         <item column="1" row="0">
          <widget class="KisIntParseSpinBox" name="intX">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Horizontal Position</string>
           </property>
           <property name="suffix">
            <string> px</string>
           </property>
           <property name="maximum">
            <number>30000</number>
           </property>
          </widget>
         </item>
         <item column="1" row="1">
          <widget class="KisIntParseSpinBox" name="intY">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Vertical Position</string>
           </property>
           <property name="suffix">
            <string> px</string>
           </property>
           <property name="maximum">
            <number>30000</number>
           </property>
          </widget>
         </item>
         <item column="1" row="3">
          <widget class="QCheckBox" name="boolGrow">
           <property name="toolTip">
            <string>Allow growing the image by dragging outside bounds of the image</string>
           </property>
           <property name="text">
            <string>Grow</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item column="2" row="3">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item column="2" row="1">
        <widget class="QLabel" name="heightLabel">
         <property name="toolTip">
          <string>Height</string>
         </property>
         <property name="text">
          <string>Height:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item column="4" row="1">
        <widget class="KisIntParseSpinBox" name="intHeight">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Height</string>
         </property>
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>30000</number>
         </property>
        </widget>
       </item>
       <item column="4" row="0">
        <widget class="KisIntParseSpinBox" name="intWidth">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Width</string>
         </property>
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>30000</number>
         </property>
        </widget>
       </item>
       <item column="4" row="2">
        <widget class="KisDoubleParseSpinBox" name="doubleRatio">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Aspect ratio</string>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="precision" stdset="0">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item column="2" row="2">
        <widget class="QLabel" name="aspectLabel">
         <property name="text">
          <string>Ratio:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item column="2" row="0">
        <widget class="QLabel" name="widthLabel">
         <property name="toolTip">
          <string>Width</string>
         </property>
         <property name="text">
          <string>Width:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item column="5" row="0">
        <widget class="QPushButton" name="lockWidthButton">
         <property name="toolTip">
          <string>Lock the width</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item column="5" row="1">
        <widget class="QPushButton" name="lockHeightButton">
         <property name="toolTip">
          <string>Lock the height</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item column="5" row="2">
        <widget class="QPushButton" name="lockRatioButton">
         <property name="toolTip">
          <string>Lock the aspect ratio</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item column="1" row="2">
      <widget class="QComboBox" name="cmbType">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Image</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Layer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Frame</string>
        </property>
       </item>
      </widget>
     </item>
     <item column="1" row="4">
      <widget class="QPushButton" name="bnCrop">
       <property name="text">
        <string>&amp;Crop</string>
       </property>
      </widget>
     </item>
     <item column="0" row="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Applies to:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item column="1" row="3">
      <widget class="QComboBox" name="cmbDecor">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thirds</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fifths</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Passport photo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Crosshair</string>
        </property>
       </item>
      </widget>
     </item>
     <item column="0" row="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Decoration:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item column="1" row="5">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="SpecialSpacer" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisIntParseSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kis_int_parse_spin_box.h</header>
  </customwidget>
  <customwidget>
   <class>KisDoubleParseSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>kis_double_parse_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>intX</tabstop>
  <tabstop>intY</tabstop>
  <tabstop>intWidth</tabstop>
  <tabstop>intHeight</tabstop>
  <tabstop>cmbType</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>